#takeoff: 4
#landing: 6
#gohome:  1
uint8 task
---
bool result
